src/elements/content-uploader/ContentUploader.js

```diff
@@ -684,11 +688,11 @@ class ContentUploader extends Component<Props, State> {
         const { overwrite, rootFolderId } = this.props;
         const { api, file, options } = item;
 
-        if (this.numItemsUploading >= UPLOAD_CONCURRENCY) {
+        if (this.uploadConcurrency + item.api.concurrency > MAX_UPLOAD_CONCURRENCY) {
```

Comment:
`api` is already destructured here, can also use that. Maybe while you're at it, could also destructure `concurrency`